runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -68,6 +61,14 @@
 import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 
+import javax.annotation.Nullable;
```

Comment:
Revert these import changes

Reply:
Hmm this is something that my IDE changed. Is there a linter or anything available that keeps the code in the style that runelite wants?

Reply:
https://github.com/runelite/runelite/wiki/Code-Conventions we have link to XML that you can just import to IntelliJ and then it will format the imports correctly